File: CPKINGDOM/CPKINGDOM.Portal/ClientApp/src/app/models/inventory.ts
export class Inventory {
    id: number;
    itemId: number;
    supplierId: number;
    costPrice: number;
    dateReceived: string;
    qtyReceived: number;
    qtyAvailable: number;
    remarks: string;
    barcode: string;
    itemName: string;
    description: string;
    srp: number;
    supplierName: string;
    categoryName: string;
    brandName: string;
    itemFullName: string;
    qtyPurchased: number;
    amountPaid: number;
    tranBodyId: number;
    isService: boolean;
    reorderPoint: number;
    isCritical: string;
    constructor() {
        this.id = 0;
        this.itemId = 0;
        this.supplierId = 0;
        this.costPrice = 0;
        this.dateReceived = '';
        this.qtyReceived = 0;
        this.qtyAvailable = 0;
        this.remarks = '';
        this.barcode = '';
        this.itemName = '';
        this.description = '';
        this.srp = 0;
        this.supplierName = '';
        this.categoryName = '';
        this.brandName = '';
        this.itemFullName = '';
        this.qtyPurchased = 0;
        this.amountPaid = 0;
        this.tranBodyId = 0;
        this.isService = false;
        this.reorderPoint = 0;
        this.isCritical = '';
    }
}

File: CPKINGDOM/CPKINGDOM.Core/Models/Inventory.cs
namespace CPKINGDOM.Core.Models
{
    public class Inventory
    {
        public int Id { get; set; }
        public int ItemId { get; set; }
        public int SupplierId { get; set; }
        public decimal CostPrice { get; set; }
        public string DateReceived { get; set; }
        public int QtyReceived { get; set; }
        public int QtyAvailable { get; set; }
        public string Remarks { get; set; }
        public string Barcode { get; set; }
        public string ItemName { get; set; }
        public string Description { get; set; }
        public decimal Srp { get; set; }
        public string SupplierName { get; set; }
        public string CategoryName { get; set; }
        public string BrandName { get; set; }
        public string ItemFullName { get; set; }
        public decimal AmountPaid { get; set; }
        public int QtyPurchased { get; set; }
        public int TranBodyId { get; set; }
        public bool IsService { get; set; }
        public string Notes { get; set; }
        public int ReorderPoint { get; set; }
        public string IsCritical { get; set; }
    }
}
